packages/core/docs/getting-started.stories.mdx

```diff
@@ -24,7 +24,7 @@ Preprocessor like Sass/Less or reference the CSS directly in your HTML.
 ```scss
 @import '~normalize.css/normalize.css'; // css reset
 @import '~@clr/core/global.min'; // clarity global styles
-@import '~@clr/core/styles/module.shims.min'; // non-evergreen browser shims (IE11, Safari)
+@import '~@clr/core/styles/module.shims.min'; // non-evergreen browser shims
```

Comment:
It sounds like this should be removed? We only support evergreen browsers after this. Right?

Reply:
No because it still has some hacks for safari in it. Once safari releases its new version, I think we might be able to get rid of it.